templates/gcp/thank-you.html

```diff
@@ -2,49 +2,51 @@
 
 {% block title %}Thank you | GCP{% endblock %}
 
-{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+{% block head_extra %}<meta name="robots" content="noindex" />{% endblock %}
 
 {% block content %}
-<div class="p-strip is-bordered is-deep">
-  <div class="row">
-    <div class="col-7">
-      <h1>Thank you for enquiring about Ubuntu on GCP</h1>
-      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-    </div>
-    <div class="col-5 u-vertically-center u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg",
-          alt="",
-          width="200",
-          height="200",
-          hi_def=True,
-          loading="auto"
-        ) | safe
-      }}
+
+  <div class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="u-no-margin--bottom">Thank you for enquiring about Ubuntu on GCP</h1>
+        <h2>A member of our team will be in touch within one working day.</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-container is-cover u-hide--small">
+          <img class="p-image-container__image"
+               src="https://assets.ubuntu.com/v1/b0fa63e9-christina-wocintechchat-com-LQ1t-8Ms5PY-unsplash.png"
+               alt="" />
+        </div>
+      </div>
     </div>
   </div>
-</div>
 
-{% include "shared/_thank_you_footer.html" %}
+  {% include "shared/_thank_you_footer.html" %}
 
 {% endblock content %}
+
 {% block footer_extra %}
-<!-- Google Code for Services Canonical &mdash; contact us Conversion Page -->
-<script>
-  /* <![CDATA[ */
-  var google_conversion_id = 1012391776;
-  var google_conversion_language = "en";
-  var google_conversion_format = "3";
-  var google_conversion_color = "ffffff";
-  var google_conversion_label = "-mBBCIC5vwMQ4L7f4gM";
-  var google_conversion_value = 0;
-  /* ]]> */
-</script>
-<script type="text/javascript" src="https://www.googleadservices.com/pagead/conversion.js"></script>
-<noscript>
-  <div style="display:inline;">
-    <img height="1" width="1" style="border-style:none;" alt="" src="https://www.googleadservices.com/pagead/conversion/1012391776/?value=0&amp;label=-mBBCIC5vwMQ4L7f4gM&amp;guid=ON&amp;script=0"/>
-  </div>
-</noscript>
+  <!-- Google Code for Services Canonical &mdash; contact us Conversion Page -->
+  <script>
+    /* <![CDATA[ */
+    var google_conversion_id = 1012391776;
```

Comment:
I will just assume this data is correct

Reply:
The only thing that was on the ticket was to update the layout, so I also have to assume that it's correct 😅